omod/src/main/webapp/pages/registerPatient.gsp

```diff
@@ -137,6 +137,10 @@ fieldset[id\$="-fieldset"] div > div {
                         if (NavigatorController.getQuestionById(questionName) != undefined) {
                             NavigatorController.getQuestionById(questionName).questionLi.addClass("done");
                         }
+                        if (fieldName == 'mother-field') {
```

Comment:
I forget, where does this name "mother-field" get set up? Is it something in our config?

Reply:
Yes, this field gets configured in our pihcore SL config.

Reply:
Got it, thanks @cioan !  I'm struggling with what the exact comment is, but can we add something here noting that this is a rather specific hack for when having a "mother-field" configured?